doc/github_actions.rst

```diff
@@ -0,0 +1,35 @@
+Integration with GitHub Actions
+===============================
+
+`Universum` requires no special integration with `GitHub Actions <https://docs.github.com/en/actions>`_. It is usually
+launched as one long step in a single build stage.
+
+.. warning::
+
+    GitHub Actions doesn't support `report_artifacts` key in configuration.
```

Comment:
Do we have an exception for the case user passes this key?

Reply:
no, we just provide a broken link

Reply:
Please clarify it in this warning then.